src/app/modules/payments/payments.service.ts

```diff
@@ -242,6 +242,7 @@ export const performPaymentPostSubmissionActions = (
                 performEncryptPostSubmissionActions({
                   submission,
                   responses: payment.responses,
+                  emailFields: [], // TODO [EMAIL-CONFIRMATION-BUG]: Email confirmation email to email fields does not work for payment forms, this is an existing issue to be fixed.
```

Comment:
agree that we can resolve this later.
- it's not a regression
- and payment forms send the users a confirmation through a different route